README.md

```diff
@@ -1,4 +1,4 @@
-# Google Cloud Trace for Node.js
+# StackDriver Trace for Node.js
```

Comment:
'StackDriver Trace` appears to be correct looking at their marketing materials.  :shrugs: